_sass/components/popovers/popover-search.scss

```diff
@@ -0,0 +1,72 @@
+.popover-search {
+  border: 0;
+  bottom: 0;
+  height: auto;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  position: relative;
+
+  &:not(.lunr-hidden) {
+    background: #fff;
+    max-height: 349px;
+    transition: all .10s ease-in-out;
+    visibility: visible;
+    width: 100%;
+
+    @include media(xs) {
+      margin-top: 80px;
+      max-width: 340px;
+    }
+  }
+
+  .result {
+    border-bottom: 1px solid $gray-lighter;
+    transition: background .075s ease-in-out;
+
+    &:hover {
+      background: rgba($gray-lighterest, .5);
```

Comment:
This state is too subtle for me, can we have an active state for touch devices too?